engines/bops_api/app/views/bops_api/v2/public/_show.json.jbuilder

```diff
@@ -1,13 +1,17 @@
 # frozen_string_literal: true
 
+# json.key_format! camelize: :lower
```

Comment:
Just need to uncomment this line when we can drop `full_reference` and `received_at`